test_suite/lris_blue_long_600_4000_d560.red

```diff
@@ -8,7 +8,7 @@ out overwrite True
 out sorted lris_blue_long_600_4000_d560
 
 # Reduce
-reduce usebias bias      # How to subtract the detector bias (bias, overscan, dark, none), you can also specify a master calibrations file if it exists.
+reduce usebias overscan   # How to subtract the detector bias (bias, overscan, dark, none), you can also specify a master calibrations file if it exists.
```

Comment:
This line can be removed, since this is the default for armlsd (as written in the doc/bias.rst file above).